pkgs/by-name/po/poetry/unwrapped.nix

```diff
@@ -77,9 +70,11 @@ buildPythonPackage rec {
       cleo
       dulwich
       fastjsonschema
+      findpython
       installer
       keyring
       packaging
+      pbs-installer
```

Comment:
We also need `pbs-installer.optional-dependencies.download` and `pbs-installer.optional-dependencies.install`.

Reply:
Added both of those. I hope I've added them correctly 🙂

Reply:
Use `++` at the end of the list.

Reply:
Hopefully now it is correct 😄